yesod-core/test/YesodCoreTest/Exceptions.hs

```diff
@@ -41,7 +41,7 @@ case500 :: IO ()
 case500 = runner $ do
     res <- request defaultRequest
     assertStatus 500 res
-    assertBody "FOOBAR" res
+    assertBodyContains "FOOBAR" res
```

Comment:
In ghc-8.0, `error` includes a call stack and the default `Show` instance for exceptions prints the callstack.

```> ghci -XScopedTypeVariables
GHCi, version 8.0.0.20160411: http://www.haskell.org/ghc/  :? for help
Loaded GHCi configuration from /home/erikd/.ghci
ghci > import Control.Exception
ghci > catch (error "FOOBAR") (\(e :: SomeException) -> print e)
FOOBAR
CallStack (from HasCallStack):
```